content/en/real_user_monitoring/guide/proxy-rum-data.md

```diff
@@ -74,16 +74,29 @@ To successfully proxy request to Datadog:
 2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
 4. The request body must remain unchanged.
 
-**IMPORTANT**: Ensure the `ddforward` attribute points to a valid RUM endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
+**IMPORTANT**: Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failing to do so may expose your systems to SSRF/XSS attacks. The list of valid intake URL patterns per sites are:
 
-| Site    | Valid RUM URL Pattern                          | Site Parameter (SDK [initialization parameter][3])| 
+For latest SDK version:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Maybe use tabs to have a more succinct output ([syntax example](https://github.com/DataDog/browser-sdk/pull/1648/commits/1591aecb5e70abe72c7dfe01000258a42049dc6e#diff-a9c53af1f3e9e5b103b5bdef39b52eac991688984575a03139b6fcfdf143ecc5R40))